PixaView/Controller/PixaViewController.swift

```diff
@@ -0,0 +1,276 @@
+//
+//  PixaViewController.swift
+//  PixaView
+//
+//  Created by Дмитрий Игнатьев on 27.11.2021.
+//
+
+import UIKit
+import TestFramework
+
+class PixaViewController: UIViewController{
+    
+    //MARK: Outlets -
+    @IBOutlet weak var collectionView: UICollectionView!
+    
+    
+    //MARK:  Properties -
+    var hitsRESULT: [Hits] = [] //array from json
+    var enumValue = ""
+    var order: Order = .popular
+    var category: [Categories] = [.computer]
+    var currentPage = 1
+    var total = 0
+    var collectionViewFlowLayout = UICollectionViewFlowLayout()
+    //MARK: - end of Properties
+    
+    
+    
+    //MARK:  viewDidLoad -
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        let sortButton = navigationItem.rightBarButtonItem = UIBarButtonItem(image: UIImage(systemName: "line.3.horizontal.circle"), style: .plain, target: self, action: #selector(pixaSortButton))
```

Comment:
You could replace it with 

```
navigationItem.rightBarButtonItem = UIBarButtonItem(
    image: UIImage(systemName: "line.3.horizontal.circle"),
    style: .plain,
    target: self,
    action: #selector(pixaSortButton)
)
```

to make it shorter. Also you don't use sortButton variable, pls remove it.
It it not good practice to make double assignment, pls avoid it.

Reply:
done